updates/stable.json

```diff
@@ -1,25 +1,25 @@
 {
   "stream": "stable",
   "metadata": {
-    "last-modified": "2020-06-30T16:26:58Z"
+    "last-modified": "2020-07-10T21:25:40Z"
   },
   "releases": [
     {
-      "version": "32.20200601.3.0",
+      "version": "32.20200615.3.0",
       "metadata": {
+        "barrier": {
+          "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/484"
+        },
         "rollout": {
           "start_percentage": 1.0
```

Comment:
Do we need to leave the rollout/start_percentage information here? I'm comparing it to `next.json` and I don't see that in there.

Reply:
Maybe not.  Let's try it.

Reply:
Yup, looks good without.